docs/topics/appendix-proficiency-levels.adoc

```diff
@@ -0,0 +1,21 @@
+
+[[proficiency_levels]]
+= Proficiency Levels
+
+Using each mission available on {launcher} requires certain minimum knowledge from you, and provides a certain information to you. For easier understanding, the requirements and concepts are organized in several levels. In addition to the levels described in this chapter, there can be additional requirements with each mission, specific to its aim or the technologies it uses.
+
+[discrete]
+== {proficiency-foundational}
+
+The missions rated at foundational proficiency generally require no prior knowledge of the subject matter; they provide general awareness and demonstration of key elements, concepts, and terminology. There are no special requirements on you except those directly mentioned in the description of the given mission.
```

Comment:
Make sure you are using {proficiency-foundational} in every place you are taking about the level. This will make it easier if you change the name.